bazel/repositories.bzl

```diff
@@ -79,11 +79,13 @@ def _envoy_repo_impl(repository_ctx):
     constraints of a `genquery`.
 
     """
-    repo_path = repository_ctx.path(repository_ctx.attr.envoy_version)
-    version = repository_ctx.read(repo_path.dirname.get_child(repo_path.basename)).strip()
-    repository_ctx.file("version.bzl", "VERSION = '%s'" % version)
-    repository_ctx.file("path.bzl", "PATH = '%s'" % repo_path.dirname)
-    repository_ctx.file("__init__.py", "PATH = '%s'\nVERSION = '%s'" % (repo_path.dirname, version))
+    repo_version_path = repository_ctx.path(repository_ctx.attr.envoy_version)
+    api_version_path = repository_ctx.path(repository_ctx.attr.envoy_api_version)
+    version = repository_ctx.read(repo_version_path).strip()
```

Comment:
It looks like the old code calls `repo_path.dirname.get_child(repo_path.basename)` where as the new code just uses `repo_version_path`. Was the old code unnecessarily complicated?